notebooks/265-qrcode-monster/265-qrcode-monster.ipynb

```diff
@@ -0,0 +1,971 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    controlnet_inputs = {
</pre><p>does it really necessary to use 768x768 input for generation? smaller input size requires less memory for conversion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
yes, it does. with 512x512 resolution I could not get a readable qr code even with the maximal controlnet guidance, 
![see the example](https://github.com/openvinotoolkit/openvino_notebooks/assets/76463150/65122be9-d580-471f-947b-7f14c0b9d5ab)